docs/reference/manifest/extensionpoint.md

```diff
@@ -23,71 +23,111 @@ For more information, see [Version overrides in the manifest](../../develop/add-
 
 |  Attribute  |  Required  |  Description  |
 |:-----|:-----|:-----|
-|  **xsi:type**  |  Yes  | The type of extension point being defined.|
+|  **xsi:type**  |  Yes  | The type of extension point being defined. Possible values depend on the Office host application; that is, the grandparent **Host** element value.|
 
-## Extension points for Excel only
+## Extension points for Word, Excel, PowerPoint, and OneNote add-in commands
 
-- **CustomFunctions** - A custom function written in JavaScript for Excel.
+There are three types of extension points available in some or all of these hosts.
 
-[This XML code sample](https://github.com/OfficeDev/Excel-Custom-Functions/blob/master/manifest.xml) shows how to use the **ExtensionPoint** element with the **CustomFunctions** attribute value, and the child elements to be used.
+- [PrimaryCommandSurface](#primarycommandsurface) (Valid for Word, Excel, PowerPoint, and OneNote) - The ribbon in Office.
+- [ContextMenu](#contextmenu) (Valid for Word, Excel, PowerPoint, and OneNote) - The shortcut menu that appears when you right-click in the Office UI.
```

Comment:
Saying right-click seems very mouse-centric. Do we need to add long-press or similar keyboard or touchscreen action? Perhaps there's a neutral/generic description of this action. The [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28838) recommends "select and hold (or right-click)"

Here and other instances